_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,260 @@
+
+The beta dramatically reduced per transaction costs for councils, completely eliminating the need to courier paper applications between councils and the DIA. Councils have communicated their intention to actively promote the online option (if it proceeds beyond beta) based on the improved experience for applicants (and lower processing costs).
+
+The statutory declaration remains the single biggest barrier preventing large scale adoption of an online application process (as well as a significant barrier to those applying on paper). Removing the statutory declaration removes the requirement to visit a council to submit an application. A [paper](https://docs.google.com/document/d/11gccvNrVD4PnfvMZwgwHEO4M5UQmRvjbpTZjBVA1lD8/edit?usp=sharing) was written to support the case for this change during the beta phase. If the case for change is supported, this requirement could be removed by June 2020.
+
+Together with the [Rates Rebate Beta Research Report](https://docs.google.com/document/d/1zDiJs8x2r9ABKa0hqCHkDZnK1B5iy-JlNSbtCZUoDuo/edit?usp=sharing), this report documents the key results of the trial, presents options for next steps, and makes a set of recommendations.
+
+# High level recommendations 
+A set of recommendations beyond beta are also included later in the document.
+
+**Run a second trial with the existing three councils during a high application volume period and implement data export solutions**
+1. A second trial should be run during a high volume period (e.g. August and September) to fully inform DIA and Council decisions about next steps. While running the beta in June meant that council staff were available to collaborate and co-design improvements to the application and process, it also meant that application volumes were lower for councils than they would have been in July and August. 
+Evidence gathered during the Alpha (which ran through the busy period) suggests that uptake of the online option is high when it is promoted by councils (20% of the 800 recipients of a letter sent by Tauranga City Council promoting the online option opted to use it in 2018). 
+The councils involved in the trial have enthusiastically requested that the trial be extended through a high volume period. 
+1. The second trial should be limited to the same three councils who participated in June/July. This will enable valuable data to be gathered while keeping the cost of the extension minimal.
+1. Delay the decision about whether to proceed beyond this extension until the end of this second trial. This will allow time for results from the first round to be validated against higher application volumes. 
```

Comment:
```suggestion
1. Delay the decision about whether to proceed beyond this extension until the end of this second trial. This will allow time for results from the first round to be validated against higher application volumes.
```